jcl/src/openj9.criu/share/classes/org/eclipse/openj9/criu/CRIUSupport.java

```diff
@@ -222,6 +228,42 @@ public CRIUSupport setFileLocks(boolean fileLocks) {
 		return this;
 	}
 
+	/**
+	 * Controls whether to dump file locks.
```

Comment:
Comment looks like a copy/paste error.  Is this the intention:
```suggestion
	 * Controls whether to re-establish TCP connects
```